packages/mds-db/geographies.ts

```diff
@@ -23,18 +24,11 @@ export async function readSingleGeography(geography_id: UUID): Promise<Geography
   }
 }
 
-export async function readGeographies(params?: {
-  get_read_only?: boolean
-}): Promise<Geography[]> {
-  // use params to filter
-  // query on ids
-  // return geographies
+export async function readGeographies(params: Partial<ReadGeographiesParams> = {}): Promise<Geography[]> {
```

Comment:
Would still prefer to see:
```
const { get_read_only } = { get_read_only: false, ...params }
```
To a) establish explicit default value, and b) obviate the need to use `params && params.get_read_only` below.

Reply:
I already did that 🤔